src/Energon.php

```diff
@@ -44,26 +44,48 @@ public static function generate($prime = null)
     }
 
     /**
+     * Generates a set of numbers ready for use (with a custom max cap).
+     *
+     * @param int $max
+     *
+     * @return array
+     */
+    public static function generateMax($max)
```

Comment:
Not sure if a lot of people are using `generate` with a predefined prime. Maybe we can release a version without that option and replace it with `$max`.